monitoring/kafka/alerts.yaml

```diff
@@ -121,7 +121,7 @@ groups:
         Kafka consumer lag has been more more than ${maxConsumerLagSecondsWarningThreshold} seconds
         or more than ${maxConsumerLagMessagesWarningThreshold} messages for 5 minutes.
 
-        Current time lag is {{ with printf 'kafka_consumergroup_group_max_lag_seconds{namespace="${namespace}",cluster_name="${cluster}",group="%s"}'
+        Current time lag is {{ with printf "kafka_consumergroup_group_max_lag_seconds{namespace='${namespace}',cluster_name='${cluster}',group='%s'}"
```

Comment:
outer double-quote is needed because we are in a Go template (so it is a string).
according to proemtheus docs, single and double quotes are the same : so we can indeed use the inner single quotes ; but it kind of looks different than all the other prometheus expressions in this file (and mostly everywhere else... So not sure if we should strive for uniformity or simplicity :-/

Reply:
Escaped the inner quotes for simplicity's sake